src/textchannellistener.cpp

```diff
@@ -1044,6 +1044,13 @@ TextChannelListener::DeliveryHandlingStatus TextChannelListener::handleDeliveryR
     int deliveryStatus = status.value<int>();
     qCDebug(lcCommhistoryd) << "[DELIVERY] Message delivery status: " << deliveryStatus;
 
+    if ((deliveryStatus == Tp::DeliveryStatusRead)
+        && (event.direction() == CommHistory::Event::Inbound)) {
+
+        NotificationManager::instance()->removeConversationNotifications(event.recipients().value(0));
```

Comment:
Hm, this now uses the default value for the second parameter, ChatTypeP2P, but this should happen here regardless of the conversation context, right?

Should here maybe be extra checks that m_GroupHandleType is not Tp::HandleTypeNone or Tp::HandleTypeRoom which seems elsewhere here meaning assumption of p2p?

Could then pass the type parameter explicitly too.